packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -753,25 +778,7 @@ export class GarbageCollector implements IGarbageCollector {
          */
         if (this.activeConnection() && !this.isSummarizerClient && this.shouldRunGC) {
             this.initializeBaseStateP.catch((error) => {
-                const dpe = DataProcessingError.wrapIfUnrecognized(
-                    error,
-                    "FailedToInitializeGC",
-                );
-                dpe.addTelemetryProperties({
-                    gcConfigs: JSON.stringify({
-                        gcEnabled: this.gcEnabled,
-                        sweepEnabled: this.sweepEnabled,
-                        runGC: this.shouldRunGC,
-                        runSweep: this.shouldRunSweep,
-                        writeAtRoot: this._writeDataAtRoot,
-                        testMode: this.testMode,
-                        sessionExpiry: this.sessionExpiryTimeoutMs,
-                        inactiveTimeout: this.inactiveTimeoutMs,
-                        trackGCState: this.trackGCState,
-                        ...this.gcOptions,
-                    }),
-                });
-                throw dpe;
+                throw error;
```

Comment:
`.catch((e) => { throw e; })` is a no-op.  It just wraps the one rejected promise in another.  But the throw still goes nowhere unless you await that promise.

Reply:
So for main containers that don't run GC, this will result in an unhandled promise rejection.  For Summarizers, it may also result in an unhandled promise rejection but eventually it will be handled - the first time GC runs it'll pop out there.

Reply:
You probably just want to do `.catch((e) => {})`, or maybe log if the error wouldn't get logged elsewhere.